### PR TITLE
Set throttling level when creating an API.

### PR DESCRIPTION
**Pull request by Meruja**
Resolving issue #6859

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java [hunk 1]
```diff
@@ -459,7 +459,7 @@ private API prepareToCreateAPIByDTO(APIDTO body) throws APIManagementException {
         assignLabelsToDTO(body,apiToAdd);
 
         // set default API Level Policy
-        if (StringUtils.isBlank(apiToAdd.getApiLevelPolicy())) {
+        if (apiToAdd.getApiLevelPolicy() != null) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java]:
```suggestion
        if (StringUtils.isEmpty(apiToAdd.getApiLevelPolicy())) {
```

How about this?

Comment (malinthaprasan) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java]: I'm a bit confused. The difference between isBlank and isEmpty is isEmpty returns false for strings with only spaces and isBlank returns true in that case. So are we differentiate between { "apiLevelPolicy": "" .. } and  { "apiLevelPolicy": " " .. } ?

**Comment by AmaliMatharaarachchi** [comment]
@Meruja please update the PR description with the correct issue. https://github.com/wso2/product-apim/issues/6859

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java (hunk 1 above)
Comment (chashikajw) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:462]: @Meruja is this solved ?

Comment (Meruja) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:462]: Yes, the entire logic has been changed with 05d94d28629d33b021fad9cf209a099633302d11

**Pull request merged**